src/components/TextInput/BaseTextInput.js

```diff
@@ -302,9 +302,11 @@ class BaseTextInput extends Component {
                     This Text component is intentionally positioned out of the screen.
                 */}
                 {this.props.autoGrow && (
+
```

Comment:
```suggestion
```

Reply:
@pecanoro If i remove this `line break` lint throws [error](https://github.com/Expensify/App/pull/9102#:~:text=If%20i%20add%20comment%20on%20top%20of%20onLayout%20between%20style%20line%20break) a line break is needed before comment.

Reply:
Really? What an odd place to add the line break lol Anyways, if it pleases the linter, all good.